### PR TITLE
#163148988 Sort Fetched Data Alphabetically

### PR DESCRIPTION
**Pull request by nakatuddesuzan**
#### What does this PR do?
Sort all fetched data alphabetically in pagination class
 - sort fetched room names alphabetically
- sort fetched block names alphabetically
- sort fetched location names alphabetically
- sort fetched office names alphabetically
- sort fetched floor names alphabetically

#### Description of Task to be completed?
*As an admin
When I query for any data 
Then the data should be returned in an alphabetical format*

#### How should this be manually tested?
- checkout branch `ft-sort-fetched-data-alphabetically-163148988`
- run  the app using  the command `make run-app` then,
- try fetching different data say offices from the database

#### What are the relevant pivotal tracker stories?
[#163148988 ](https://www.pivotaltracker.com/story/show/163148988)
#### Screenshots
screen shot 2019-01-23 at 21 30 57
screen shot 2019-01-23 at 21 32 44
screen shot 2019-01-23 at 21 34 36
screen shot 2019-01-23 at 21 35 22
screen shot 2019-01-23 at 21 37 14
screen shot 2019-01-23 at 21 37 50
screen shot 2019-01-23 at 21 38 27
screen shot 2019-01-23 at 21 39 39
screen shot 2019-01-23 at 21 40 27
screen shot 2019-01-23 at 21 40 46
screen shot 2019-01-23 at 21 41 21
screen shot 2019-01-23 at 21 44 45
screen shot 2019-01-23 at 22 01 54
screen shot 2019-01-23 at 22 04 45
screen shot 2019-01-23 at 22 05 24
screen shot 2019-01-23 at 22 07 01

### CODE REVIEW

**Comment by kampkelly** [comment]
Nice work!

**Comment by cwizard2011** [comment]
Nice work, just try to implement @kampkelly feedback and I think you are good

**Comment by nakatuddesuzan** [comment]
@Bonifase , Thanks a lot. The screenshots are many but I have Included most of them now

File: api/devices/schema.py [hunk 1]
```diff
@@ -62,7 +62,7 @@ class Query(graphene.ObjectType):
 
     def resolve_all_devices(self, info):
         query = Devices.get_query(info)
-        return query.all()
+        return query.order_by(DevicesModel.name).all()
```
Comment (omobosteven) [inline api/devices/schema.py]: is there a reason this is not sorted using the lower function?

Comment (nakatuddesuzan) [reply api/devices/schema.py]: @omobosteven , There is no reason. I just missed modifying that part. Thanks for pointing that out. It is all done now

**Pull request merged**